.tito/custom.py

```diff
@@ -0,0 +1,62 @@
+#
+# Copyright (c) 2016 Red Hat, Inc.
+#
+# All rights reserved.
+#
+# Author: Jan Pokorny <jpokorny@redhat.com>
+# (using src/tito/builder/fetch.py from the tito project as a template)
+#
+# This source code is licensed under the GNU Lesser General Public License
+# version 2.1 or later (LGPLv2.1+) WITHOUT ANY WARRANTY.
+#
+
```

Comment:
Actually, programatically accessible authorship is at least practically
usable -- in clufter, I anticipated new contributors and wanted to make
them automatically acknowledged in "clufter --version" and friends.

Granted, the authorship will get naturally convoluted in time and that's
where VCS gets handy (that's the real final answer), but still, to have
origins and significant rewrites captured directly in the file is IMHO
worth line or two in the file itself.

Another thing missing in the proposed template is a "coding" line amongst
first 2? in the file that will allow for smooth handling of non-ASCII
characters in the file, at least in Python 2 case.

That being said, that's what's present in current attempt to cater
the guidelines...